src/setupTests.js

```diff
@@ -43,3 +44,18 @@ global.mount = mount;
 global.shallow = shallow;
 global.React = React;
 global.ReactDOM = ReactDOM;
+
+// Set up a global fetch mock, and assert that all calls to it are expected.
+beforeEach(() => {
+  // Mock fetch, and don't expect any requests. All requests will be unmatched,
+  // unless individual tests add more specific matches.
+  fetchMock.catch({ status: 500 });
+});
+
+afterEach(() => {
+  // Fail if any unmatched requests were made
+  const unmatchedCalls = fetchMock.calls(false);
+  expect(unmatchedCalls).toEqual([]);
```

Comment:
i wish jest had a way to print a custom error message like chai.

Reply:
I read a bunch of GH issues. It seems the suggested way is to either wrap individual assertions in calls to `test`/`it`, or to write specific matchers to have better messages. I'll experiment with the first one. It might make sense for assertions that don't make enough sense alone.

Reply:
Unfortunately, it seems you can't put `describe` or `test` blocks in an `afterEach`. They just silently don't run.

Reply:
😞 oh well